src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.scss

```diff
@@ -192,3 +192,52 @@ button {
   text-decoration: underline;
   padding-top: 1px;
 }
+
+.whole_changelog_wrapper {
+  max-width: 800px;
+  width: 90vw;
+  display: block;
+
+  .banner {
+    top: 0;
+    left: 0;
+    right: 0;
+    padding: 20px;
+    box-sizing: border-box;
+    position: absolute;
+    background: $light-blue-gradient;
+
+    &:after {
+      content: '';
+      background: white;
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      opacity: 0.04;
+      mask: $cube-pattern repeat;
+    }
+
+    .banner__title {
+      color: white;
+      font-size: 24px;
+      font-weight: bold;
+    }
+
+    .banner__sub {
+      color: white;
+      opacity: 0.85;
+      font-size: 16px;
+    }
+  }
+
+  .changelog_wrapper {
+    margin-top: 100px;
+  }
+
+}
+
+clark-generic-collection-logo {
```

Comment:
Most of the changes in this file are due to formatting. This is an actual change.